### DISCUSSION TITLE
How to visualize the training?

### DISCUSSION THREAD

**Discussion by arundhati08mishra** [AlexeyAB/darknet#7802] [Q&A]
To visualize the training epochs and loss, tensorboard is used with the **`model.fit()`** method. There is no such command here. For training `./darknet detector test ./cfg/coco.data ./cfg/yolov4.cfg ./yolov4.weights` is used. How to visualize the training using tensorboard?
